Makefile

```diff
@@ -141,3 +141,23 @@ deploy-api:
 
 deploy-gl:
 	aws lambda update-function-code --function-name glGenerator --zip-file fileb://./.build/pkg/backend.zip --region $(AWS_REGION) > /dev/null
+
+# ===================================
+# Add Local Tdi Files
+# ===================================
+
+put-local-tdi17:
```

Comment:
the make command can also accept paths

Reply:
https://stackoverflow.com/a/2826069

Reply:
example: https://github.com/bcgov/hcap/blob/master/Makefile#L56

Usage: https://github.com/bcgov/hcap#to-seed-the-database-with-data-in-an-excel-sheet